src/pages/resultsView/network/files/mutationMapper.js

```diff
@@ -1112,16 +1112,19 @@ var MutationDetailsTableFormatter = (function()
 	{
 		var text = "NA";
 		var total = alt + ref;
-		var style = "";
-		var tipStyle = "";
-
-		if (frequency)
-		{
-			style = "mutation_table_allele_freq";
-			text = frequency.toFixed(2);
-			tipStyle = tipClass;
-		}
+		var style = "mutation_table_allele_freq";
+		var tipStyle = tipClass;
```

Comment:
Sorry that I just noticed this, but these variables are not necessary. Please just put their values in directly on line 1128